chain/trie/trie.go

```diff
@@ -334,7 +334,7 @@ func (t *Trie) Copy() *Trie {
 }
 
 func (t *Trie) TryTraverse() error {
-	return t.traverse(t.root, true)
+	return t.traverse(t.root, false)
```

Comment:
Why change this?

Reply:
Too many unuseful log when set `print` to true

Reply:
Then make `TryTraverse` accept a parameter because it's also used elsewhere